src/go/rpk/pkg/cli/cmd/plugin/plugin.go

```diff
@@ -166,67 +177,80 @@ be overridden by specifying the --bin-dir flag.
 		Args: cobra.ExactArgs(1),
 		Run: func(_ *cobra.Command, args []string) {
 			name := args[0]
+			var (
+				autoComplete bool
+				body         []byte
+				err          error
+			)
+			if len(version) > 0 {
+				body, err = tryDirectDownload(name, version)
+				if err != nil {
+					log.Debugf("unable to download: %v", err)
+				}
```

Comment:
We need to add autocompletion to the new plugin; however, it's ok not to set `autocomplete = true` at the moment because the message below isn't _required_. We can add autocompletion to the plugin, and then set autocomplete to true here in a follow up PR.